schema.cue

```diff
@@ -0,0 +1,62 @@
+import (
+	"time"
+)
+
+// Date is a date in the format YYYY-MM-DD
+#Timestamp: time.Format(time.RFC3339)
+
+#TypeURI: string
+
+#BaselineControlID: =~"^OSPS-[A-Z]{2}-[0-9]{2}.[0-9]{2}$"
+
+#ResourceDescriptor: {
+	description: string
+	mediaType:   string
+	uri:         #TypeURI
+}
+
+#Ella: {
+	uri:     #TypeURI
+	uriData: _ // see https://cuelang.org/docs/tour/types/top/
+}
+
+#EllaEvidence: {
+	name?: string
```

Comment:
In practice, this seems to have always been `automated-scan`, so I'd propose cutting this level of indirection.